src/System/IO/Atomic.hs

```diff
@@ -0,0 +1,49 @@
+{- © 2019 Serokell <hi@serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+
+-- | Helpers for atomic file IO.
+--
+-- This module is a crossover of the `atomic-write` and `safeio` libraries
```

Comment:
I'd much rather you give a description of what this module does than where it comes from, since that's what I'd want to know if I were reading the documentation.

Reply:
> Helpers for atomic file IO.

Not good enough? 🤔

Reply:
> This module provides functions that ensure a file is written atomically by first writing the new contents to a temporary file. It also ensures this temporary file is cleaned up when an exception is raised.
Something like that.